packages/jupyter-chat/package.json

```diff
@@ -0,0 +1,216 @@
+{
+    "name": "@jupyter/chat",
+    "version": "0.1.0",
+    "description": "A chat package for Jupyterlab extension",
```

Comment:
Maybe make it clear in the description that these are utilities for creating chats